app/uk/gov/hmrc/gform/pdfgenerator/PdfGeneratorService.scala

```diff
@@ -26,7 +26,7 @@ class PdfGeneratorService(pdfGeneratorConnector: PdfGeneratorConnector) {
 
   def generatePDF(html: String)(implicit hc: HeaderCarrier): Future[Array[Byte]] = {
     val headers = Seq((HeaderNames.CONTENT_TYPE, MimeTypes.FORM))
-    val body = Map("html" -> Seq(html))
+    val body = Map("html" -> Seq(html),"force-pdfa" -> Seq("false"))
```

Comment:
This edit is not formatted by `scalafmt`.

Please run `sbt compile` which will trigger `scalafmt` formatter for you.

Reply:
Updated PR